### ISSUE TITLE
Died while a ghost and unable to move.

### ISSUE THREAD

**Issue by Jinteki01** [chromiecraft/chromiecraft#4093]
### What client do you play on?

enUS

### Faction

Alliance

### Content Phase:

40-49

### Current Behaviour

I was in a random dungeon finder group for Zul'Furrak. I purposefully died to the mobs after the run was done so I could turn in quests in Tanaris. After I died I left the dungeon group so I would stay in the zone after resurrecting. While running back I used my newly acquired talent point from leveling to 47. Immediately after using the talent point my ghost died with a "Release Spirit" prompt. I clicked the release spirit button, but nothing happened and I could not move my character any more.
![WoWScrnShot_091922_061540](https://user-images.githubusercontent.com/60189062/190998851-9e815ab2-b416-4816-8968-696114c3ca13.jpg)

### Expected Blizzlike Behaviour

It should allow me to run back to the dungeon to recover my body regardless of my grouping status after a random dungeon finder party. I was able to successfully do this just the day before in the same dungeon. The only interaction that was different, was using my talent point. 

### Source

Typical game mechanic

### Steps to reproduce the problem

1. Die while in a random dungeon finder group inside the dungeon.
2. Release spirit and begin running ghost back to the dungeon.
3. Leave the dungeon finder party.
4. Allocate talent point.
5. Character's ghost dies with a release spirit prompt,
6. Click release spirit.

### Extra Notes

After relogging my ghost was no longer dead and was able to move again.

### AC rev. hash/commit

https://github.com/chromiecraft/azerothcore-wotlk/commit/04db1914d839060501eacaa13b6fd1e5d274677c

### Operating system

Ubuntu 20.04

### Modules

- [mod-ah-bot](https://github.com/azerothcore/mod-ah-bot)
- [mod-bg-item-reward](https://github.com/azerothcore/mod-bg-item-reward)
- [mod-cfbg](https://github.com/azerothcore/mod-cfbg)
- [mod-chat-transmitter](https://github.com/azerothcore/mod-chat-transmitter)
- [mod-chromie-xp](https://github.com/azerothcore/mod-chromie-xp)
- [mod-cta-switch](https://github.com/azerothcore/mod-cta-switch)
- [mod-desertion-warnings](https://github.com/azerothcore/mod-desertion-warnings)
- [mod-dmf-switch](https://github.com/azerothcore/mod-dmf-switch)
- [mod-duel-reset](https://github.com/azerothcore/mod-duel-reset)
- [mod-eluna](https://github.com/azerothcore/mod-eluna)
- [mod-ip-tracker](https://github.com/azerothcore/mod-ip-tracker)
- [mod-low-level-arena](https://github.com/azerothcore/mod-low-level-arena)
- [mod-low-level-rbg](https://github.com/azerothcore/mod-low-level-rbg)
- [mod-multi-client-check](https://github.com/azerothcore/mod-multi-client-check)
- [mod-progression-system](https://github.com/azerothcore/mod-progression-system)
- [mod-pvp-titles](https://github.com/azerothcore/mod-pvp-titles)
- [mod-pvpstats-announcer](https://github.com/azerothcore/mod-pvpstats-announcer)
- [mod-queue-list-cache](https://github.com/azerothcore/mod-queue-list-cache)
- [mod-rdf-expansion](https://github.com/azerothcore/mod-rdf-expansion)
- [mod-transmog](https://github.com/azerothcore/mod-transmog)
- [mod-weekend-xp](https://github.com/azerothcore/mod-weekend-xp)
- [mod-instanced-worldbosses](https://github.com/nyeriah/mod-instanced-worldbosses)
- [lua-carbon-copy](https://github.com/55Honey/Acore_CarbonCopy)
- [lua-exchange-npc](https://github.com/55Honey/Acore_ExchangeNpc)
- [lua-custom-worldboss](https://github.com/55Honey/Acore_CustomWorldboss)
- [lua-level-up-reward](https://github.com/55Honey/Acore_LevelUpReward)
- [lua-recruit-a-friend](https://github.com/55Honey/Acore_RecruitAFriend)
- [lua-send-and-bind](https://github.com/55Honey/Acore_SendAndBind)
- [lua-temp-announcements](https://github.com/55Honey/Acore_TempAnnouncements)
- [lua-zonecheck](https://github.com/55Honey/acore_Zonecheck)
- [lua-zone-debuff](https://github.com/55Honey/Acore_ZoneDebuff)

### Customizations

None

### Server

ChromieCraft

**Comment by BeefSteaker** [comment]
hi Jinteki, ive been unable to reproduce this on the PTR. 
is there any more info you can provide like logs?

**Comment by Jinteki01** [comment]
Unfortunately not, I tried to be as detailed as I could. The only other thing I can I think of is the specific point I spent on the talent tree. It was in the third rank of Combat Expertise on the protection paladin tree. This increases my stamina by a percentage, so there might have been a weird interaction there with increasing my health points while dead. If it happens again in the future on another character, I'll make sure to gather the logs as well. I do appreciate you looking into this.
